.golangci.yml

```diff
@@ -26,3 +26,9 @@ linters:
 
 issues:
   exclude-use-default: false
+
+linters-settings:
+  revive:
+    rules:
+    - name: package-comments
+      disabled: true
```

Comment:
Can you clean up this last line? Weird that the red thing is on a different line. Perhaps a Windows/Unix newline thing?

Reply:
Changes pushed, and IDE settings corrected. :+1: